fsd_config/form_jsons/hsra_r1/hsra-r1-eligibility.json

```diff
@@ -0,0 +1,694 @@
+      "title": "Is the vacancy rate in the designated high street or town centre 8% or higher?",
+      "components": [
+        {
+          "name": "Mhzypt",
+          "options": {
+            "hideTitle": true
+          },
+          "type": "YesNoField",
+          "title": "Is the vacancy rate in the designated high street or town centre 8% or higher?",
+          "schema": {}
+        }
+      ],
+      "next": [
+        {
+          "path": "/in-the-last-2-years-has-the-property-been-vacant-for-365-days-or-more",
+          "condition": "Kkqowe"
+        },
+        {
+          "path": "/you-cannot-apply-for-this-funding-XbgnHz",
+          "condition": "jPvjPM"
+        }
+      ],
+      "section": "mwumLN",
+      "controller": "./pages/continue.js"
+    },
+    {
+      "path": "/in-the-last-2-years-has-the-property-been-vacant-for-365-days-or-more",
+      "title": "In the last 2 years, has the property been vacant for 365 days or more?",
+      "components": [
+        {
+          "name": "pPNbfh",
+          "options": {
+            "hideTitle": true
+          },
+          "type": "YesNoField",
+          "title": "In the last 2 years, has the property been vacant for 365 days or more?",
+          "schema": {}
+        }
+      ],
+      "next": [
+        {
+          "path": "/do-you-expect-us-to-defray-all-your-costs-by-31-march-2025",
+          "condition": "REqbHB"
+        },
+        {
+          "path": "/you-cannot-apply-for-this-funding-XUPfWc",
+          "condition": "zwwxkY"
+        }
+      ],
+      "section": "mwumLN",
+      "controller": "./pages/continue.js"
+    },
+    {
+      "path": "/do-you-expect-us-to-defray-all-your-costs-by-31-march-2025",
+      "title": "Do you expect us to defray all your costs by 31 March 2025?",
+      "components": [
+        {
+          "name": "WJoaFR",
+          "options": {
+            "hideTitle": true
+          },
+          "type": "YesNoField",
+          "title": "Do you expect us to defray all your costs by 31 March 2025?",
+          "schema": {}
+        }
+      ],
+      "next": [
+        {
+          "path": "/summary",
+          "condition": "xIzxnn"
+        },
+        {
+          "path": "/you-cannot-apply-for-this-funding-pCiqXB",
+          "condition": "gVEhwH"
+        }
+      ],
+      "section": "mwumLN",
+      "controller": "./pages/continue.js"
+    },
+    {
+      "path": "/you-cannot-apply-for-this-funding-uvglJP",
+      "title": "You cannot apply for this funding",
+      "components": [
+        {
+          "name": "OgFgQu",
+          "options": {},
+          "type": "Html",
+          "content": "<p class=\"govuk-body govuk-!\">\nBased on your answer, you’re not eligible to apply for funding to cover the cost of delivering a high street rental auction.</p>\n",
+          "schema": {}
+        }
+      ],
+      "next": []
+    },
+    {
+      "path": "/you-cannot-apply-for-this-funding-TjEVPb",
```

Comment:
https://github.com/communitiesuk/digital-form-builder/pull/539/files#r1625914028 it seems there are few places